pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/ReplicatedSubscriptionsController.java

```diff
@@ -75,6 +76,10 @@ public class ReplicatedSubscriptionsController implements AutoCloseable, Topic.P
                     "Counter of currently pending snapshots")
             .register();
 
+    private static final Counter timedoutSnapshotsMetric = Counter
+            .build().name("pulsar_replicated_subscriptions_snapshot_timeouts")
```

Comment:
Perhaps `pulsar_replicated_subscriptions_timedout_snapshots` would be more consistent with the existing `pulsar_replicated_subscriptions_pending_snapshots`? I don't have a strong opinion about this.